src/icons/Exchange.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import React from 'react'
+
+export default ({ size, ...p }: { size: number }) => (
+  <svg viewBox="0 0 16 16" height={size} width={size} {...p}>
+    <path
```

Comment:
same, can be a const

Reply:
Maybe we should consider doing a pass on all icons to do that.